lib/view/ppe/ppe_off_guidance_2.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_svg/svg.dart';
+
+import '../../hard_data.dart';
+import '../../style.dart';
+import '../../widget/cards/ppe_step_card.dart';
+import '../../widget/notification_banner.dart';
+
+class PPEOffGuidance2 extends StatelessWidget {
+  final title = 'Method 2';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        backgroundColor: AppColors.grey50,
+        appBar: AppBar(
+          backgroundColor: AppColors.appBarBackground,
+          iconTheme: AppStyles.appBarIconTheme,
+          title: Text(
+            title,
+            style: AppStyles.textH5,
+          ),
+        ),
+        body: ListView(children: <Widget>[
+          NotificationBanner(
+            backgroundColor: AppColors.grey50,
+            icon: SvgPicture.asset(
+              'assets/images/icon/SVG/icon_warning.svg',
+            ),
+            message: ppeOffWarning,
+          ),
+          ...ppeOffMethod2Steps
```

Comment:
It would be nice to not have logic in the widgets, could this go into a separate "viewmodel" class, then its easier to have a unit test for it too.

Reply:
Like that idea... will need a general pretty big refactor so that PPE On will inherit the same model.

Will also have to change the models to something like:

```
import 'package:flutter/material.dart';

class PPEStepInfo {
  final String image;
  final String text;
  final List<String> notes;

  const PPEStepInfo({
    @required this.image,
    @required this.text,
    @required this.notes,
  });
}

class PPEStepInfoCardModel extends PPEStepInfo {
  int step;

  PPEStepInfoCardModel(
      {String image, String text, List<String> notes, this.step})
      : super(image: image, text: text, notes: notes);
}
```

Reply:
@ericjiang97 I dont really see why this requires a major refactor vs a small change - I'll discuss this with you via slack.